docs/DeveloperGuide.md

```diff
@@ -196,7 +196,20 @@ The HelpWindow dialog box provides users with the link to our User Guide and the
 
 
 ### Find Command <a name="find_command"></a>
+The Find command allows users to find a contact in their list by keywords. 
+Keywords can target contact's names & tags and/or a certain social media platform type. 
```

Comment:
Let's change this to: "A contact's name, tags and/or social media platforms can be specified as keywords."